projects/core/template.yaml

```diff
@@ -107,3 +107,38 @@ Resources:
       Tags:
         - Key: Name
           Value: !Sub "${AWS::StackName}-user_info"
+
+  SessionsTable:
```

Comment:
Only comment is that these tables (userinfo, session etc but not client registry) should be in the main template